superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -78,7 +80,7 @@ interface AlertReportModalProps {
   show: boolean;
 }
 
-const NOTIFICATION_METHODS: NotificationMethod[] = ['Email', 'Slack'];
+const DEFAULT_NOTIFICATION_METHODS: NotificationMethodOption[] = ['Email'];
```

Comment:
not sure if changing the default is a breaking change, but since we should probably never hit this code path (can the page render without the config options actually here?) it's likely fine

Reply:
if system admin didn't set slack API token in config, user did saw Slack option before, but they can't see report in slack.